docs/content/en/functions/markdownify.md

```diff
@@ -23,6 +23,8 @@ aliases: []
 {{ .Title | markdownify }}
 ```
 
-{{< new-in "0.93.0" >}} **Note**: `markdownify` now supports [Render Hooks] just like [.RenderString](/functions/renderstring/).
+{{< new-in "0.93.0" >}} **Note**: `markdownify` now supports [Render Hooks] just like [.RenderString]. However, if you use more complicated [Render Hooks] relying on Page Context, preferer [.RenderString] - see [GitHub issue #9692](https://github.com/gohugoio/hugo/issues/96920) for more details.
```

Comment:
```suggestion
{{< new-in "0.93.0" >}} **Note**: `markdownify` now supports [Render Hooks] just like [`.Page.RenderString`]. However, if you use more complicated [Render Hooks] relying on page context, use [`.Page.RenderString`] instead. See [GitHub issue #9692](https://github.com/gohugoio/hugo/issues/96920) for more details.
```

Reply:
The issue number in the link above is wrong.

Reply:
Thanks! Updated the PR